vue-frontend/src/pages/jobs/[id].vue

```diff
@@ -292,26 +291,37 @@ export default {
     },
     getJobDates() {
       var maxDays = 15;
+      const currentDate = new Date().getDate();
 
-      // current month day
-      var currentDay = moment().format("DD");
+      const firstDateOfMonth = new Date();
+      const year = firstDateOfMonth.getFullYear();
```

Comment:
Instead of assigning new variables, i think best practice is to use them directly to avoid memory consumption